apps/intellij-extension/README.md

```diff
@@ -0,0 +1,44 @@
+# Otter IntelliJ IDEA / WebStorm Plugin
+
+This package contains the IntelliJ plugin for Otter framework.
+
+## Available commands
+
+In WebStorm or IntelliJ IDEA, you can execute actions via the right click -> refactor or via any other refactor menu of the IDE.
+
+The following actions are currently available:
+
+### Extract
+
+| Action               | Description                                 | How to execute                                                      |
+|----------------------|---------------------------------------------|---------------------------------------------------------------------|
+| Extract CSS variable | Extract a SCSS property into a o3r.variable | Cursor on the line, right click -> refactor -> Extract CSS variable |
```

Comment:
Could it be right-click -> Otter -> Extract SCSS property to o3r.variable ?
to match what we have in the VSCode extension

Reply:
Not sure if it would be available in the right click but something similar would be possible I guess. I will not be able to do the change in a reasonable timeframe. I can still update the title to be the one you gave, even though it won't be under an "otter" menu. 
[Link of the info](https://plugins.jetbrains.com/docs/intellij/grouping-action.html)